electron.makefile

```diff
@@ -1,5 +1,5 @@
 # Common flags for electron-packager on all platforms
-ELECTRON_COMMON=. "Glowing Bear" --overwrite --version-string.FileDescription="Glowing Bear" --ignore=node_modules --ignore=test --ignore=bower_components
+ELECTRON_COMMON=. "Glowing Bear" --overwrite --version-string.FileDescription="Glowing Bear" --prune --ignore=test --ignore=bower_components
```

Comment:
I had to change this so that electron-packager will include the node module in the built package.